plugins/parser/multiline.md

```diff
@@ -23,20 +23,21 @@ See [Parse Section Configurations](/configuration/parse-section.md).
 Specifies the regexp pattern for the start line of multiple lines. Input plugin
 can skip the logs until `format_firstline` is matched. Default is `nil`.
 
-If `format_firstline` is not specified, input plugin should store
-unmatched new lines in temporary buffer and try to match buffered logs
+If `format_firstline` is not specified, the input plugin should store the
+unmatched new lines in the temporary buffer and try to match the buffered logs
 with each new line.
 
 
-### `formatN`
+### `formatN` (required)
```

Comment:
With this approach, link also contains `(required)`, right?
I want to avoid it. Because if we change such parameters to optional, the link will be broken.
Other same changes too.

Reply:
I believe there was a mix of both conventions in the documentation. I tried to use only this one.
However, to avoid broken links, all the `(optional)` or `(required)` attributes should be moved with the description.
I'll update those as well.

Thank you for reviewing!